tox.ini

```diff
@@ -73,7 +77,7 @@ changedir={toxinidir}
 [tox:travis]
 ; This section is used by tox-travis
 ; https://pypi.python.org/pypi/tox-travis
-2.7 = py27-coverage-posix,py27-nomodules-posix,topfile,narrativedocs,pyflakes
+2.7 = py27-coverage-posix,py27-nomodules-posix,topfile,narrativedocs,pyflakes,txchecker-travis
```

Comment:
We should really get rid of tox-travis and have a proper build matrix; but that is for a separate ticket, I think.